src/Slogan.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 
-export default function Slogan(props) {
+export default function Slogan({ slogan, key }) {
   return (
-    <div className='slogan'>
+    <li key={key} className='slogan'>
```

Comment:
Since you're assigning a Key in your SloganList, you don't need to assign a key to the li here.